services/bb-pacman/backend/business-partner-agent/src/main/java/org/hyperledger/bpa/controller/AdminController.java

```diff
@@ -84,6 +85,17 @@
         return HttpResponse.notFound();
     }
 
+    /**
+     * Aries: Create a schema configuration
+     *
+     * @param req {@link CreateSchemaRequest}
+     * @return {@link HttpResponse}
+     */
+    @Post("/schema/create")
```

Comment:
Is there already a POST to '/schema'? It feels like this should be it.

Reply:
there is already one - it's for adding a schema (import).
this is for later refactoring when we figure out exactly what we want, just trying to not change much of their code at this time and introduce new bugs.